.github/workflowsdeployment.yml

```diff
@@ -11,7 +13,27 @@ on:
 
 jobs:
 
-  build:
+  Version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: fregante/setup-git-token@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create tag if necessary
+        if: github.event_name == 'schedule'
```

Comment:
Technically this should be on the whole job, but then `Build` won't run at all since it `needs` this job.